assets/src/edit-story/karma/fixture/fixture.js

```diff
@@ -803,6 +810,40 @@ class APIProviderFixture {
 }
 /* eslint-enable jasmine/no-unsafe-spy */
 
+class CurrentUserProviderFixture {
```

Comment:
As per https://github.com/google/web-stories-wp/pull/6132#discussion_r568708249, I don't think this is needed. Seems to be a bit too much.

Only mocking the `getCurrentUser` method from the APIProvider should be needed here. That'd be enough to mock any state of the current user if needed.